packages/core/src/Stores/Helpers/client-notifications.js

```diff
@@ -412,11 +431,19 @@ const hasMissingRequiredField = (account_settings, client, isAccountOfType) => {
     }
 };
 
-const getStatusValidations = status_arr =>
+const getStatusValidations = status_arr => {
     status_arr.reduce((validations, stats) => {
```

Comment:
since added callibraces, these status were not returned
```suggestion
    return status_arr.reduce((validations, stats) => {
```